index.js

```diff
@@ -36,13 +40,21 @@ function core(value, processor) {
 }
 
 // Alex.
-function alex(value, allow) {
+function alex(value, config) {
+  let allow
```

Comment:
```suggestion
  var allow
```

Rest of the codebase uses var, which is fine for now!